### PR TITLE
Connected the DB (MySQL) to NodeJS

### PR DESCRIPTION
**Pull request by shubhmore1212**

### CODE REVIEW

File: controller/user.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { getUsers } from "../services/user.js";
+
+export const getAllUsers = async (req, res) => {
+  try {
+    const user = await getUsers();
+    res.status(200).json({ data: user[0] });
```
Comment (mayank-s-dev) [inline controller/user.js]: create a module for sending a response

File: controller/user.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  } catch (error) {
+    res.status(400).json({ error: error.message });
```
Comment (mayank-s-dev) [inline controller/user.js]: Read about status code

File: controller/user.js (hunk 1 above)
Comment (shubhmore1212) [reply controller/user.js]: Done

File: controller/user.js (hunk 2 above)
Comment (shubhmore1212) [reply controller/user.js]: Done

**Pull request merged**